stdlib/public/core/Builtin.swift

```diff
@@ -326,10 +326,33 @@ internal func _usesNativeSwiftReferenceCounting(theClass: AnyClass) -> Bool {
 #endif
 }
 
+#if os(Windows)
+// If the return type of a C/C++ function is a structure,
+// the calling convension is different from caller's.  
+// A wrapper function is introduced.
```

Comment:
Can you change `TwoWordPair::Return` in the runtime to be an `__int128` on Win64 instead of exposing the C calling convention difference in Swift code?